test/Round-test.js

```diff
@@ -143,16 +143,4 @@ describe('Round', function() {
     const round = new Round(deck);
     expect(round.endRound).to.be.a('function');
   });
-  it('endRound should return that the round has ended and percent of correct answers', function() {
```

Comment:
I couldn't find a way of testing console logs without logging within the test itself. I will go back later and test the endRound function properly.